endpoints-support.md

```diff
@@ -78,6 +78,12 @@
 |<sub>/rest/fcoe-networks/{id}</sub>                                                      | PUT      | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/fcoe-networks/{id}</sub>                                                      | DELETE   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/fcoe-networks/bulk-delete</sub>                                               | POST     | :white_check_mark:   | :white_check_mark:   |
+|     **Firmware Drivers**                                                                                                                         |
+|<sub>/rest/firmware-drivers</sub>                                                        | GET      | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:   |
```

Comment:
havent we validated this against 2400 and 2200?

Reply:
changed